features/steps/gestalt.steps.ts

```diff
@@ -1,9 +1,20 @@
-import { Then } from 'cucumber';
+import {execSync} from 'child_process';
```

Comment:
I recommend using [this API](https://github.com/gestaltjs/gestalt/blob/main/features/lib/system.ts#L13) instead, which uses [shelljs](https://www.npmjs.com/package/shelljs) under the hood to ensure compatibility across OSs. It also has support for enabling/disabling logs through env. variables:

```
DEBUG=1 pnpm run features # logs of commands are printed
```